karma.local.conf.js

```diff
@@ -2,43 +2,17 @@
 
 module.exports = function(config) {
   config.set({
-    // base path that will be used to resolve all patterns (eg. files, exclude)
-    basePath: "",
-    // frameworks to use
-    // available frameworks: https://npmjs.org/browse/keyword/karma-adapter
     frameworks: [ "mocha" ],
-    // list of files / patterns to load in the browser
-    files: [
-      "dist/jsonapi-client-test.js"
-    ],
-    // list of files to exclude
-    exclude: [ ],
-    // preprocess matching files before serving them to the browser
-    // available preprocessors: https://npmjs.org/browse/keyword/karma-preprocessor
-    preprocessors: { },
-    // test results reporter to use
-    // possible values: "dots", "progress"
-    // available reporters: https://npmjs.org/browse/keyword/karma-reporter
-    reporters: ["progress"],
-    // web server port
+    files: [ "dist/jsonapi-client-test.js" ],
+    reporters: [ "spec" ],
+    plugins: [ "karma-mocha", "karma-firefox-launcher", "karma-spec-reporter" ],
     port: 9876,
-    // enable / disable colors in the output (reporters and logs)
     colors: true,
-    // level of logging
-    // possible values: config.LOG_DISABLE || config.LOG_ERROR || config.LOG_WARN || config.LOG_INFO || config.LOG_DEBUG
     logLevel: config.LOG_INFO,
-    // enable / disable watching file and executing tests whenever any file changes
     autoWatch: false,
-    // start these browsers
-    // available browser launchers: https://npmjs.org/browse/keyword/karma-launcher
-    browsers: [ /*"Chrome", */"Firefox"/*, "PhantomJS"*/ ],
-    // Continuous Integration mode
-    // if true, Karma captures browsers, runs the tests and exits
+    browsers: [ "Firefox"/*, "PhantomJS"*/ ],
```

Comment:
PhantomJS doesn't currently support HTTP PATCH requests If/when it does, adding this back in will work.